src/styles/utils/normalize.scss

```
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
+  transition: 300ms;
+}
```

in general * selector only for box-sixing: border-box property
reset margin and set transition only for elements which need it.

```suggestion
* {
  box-sizing: border-box;
}
```